create-cluster/src/index.ts

```diff
@@ -12,6 +12,9 @@ async function run() {
     const k8sDistribution = core.getInput('kubernetes-distribution');
     const k8sVersion = core.getInput('kubernetes-version');
     const k8sTTL = core.getInput('ttl');
+    const diskGib: number = +(core.getInput('disk-gib') || 50);
+    const nodeCount: number = +(core.getInput('node-count') || 1);
```

Comment:
Why having `50` and `1`. If nothing is set, the default should be defined by vendor-api

Reply:
Agreed, will change that here. Should we change this in the CLI so we have a single source for defaults? https://github.com/replicatedhq/replicated/blob/main/cli/cmd/cluster_create.go#L31-L32